mtags/src/main/scala-2/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -129,28 +130,44 @@ trait MtagsEnrichments extends CommonMtagsEnrichments {
     def stripBackticks: String = value.stripPrefix("`").stripSuffix("`")
     def isBackticked: Boolean =
       value.size > 1 && value.head == '`' && value.last == '`'
-    def toAbsolutePath: AbsolutePath = toAbsolutePath(true)
+    def toAbsolutePath: AbsolutePath = {
+      toAbsolutePath(true)
+    }
```

Comment:
Is this change needed? To wrap it in `{}`?

Reply:
Removed